### ISSUE TITLE
Keycastr doesnt show in Security & Privacy -> Accessibility

### ISSUE THREAD

**Issue by VGraupera** [keycastr/keycastr#103]
I installed Keycastr using `brew cask install keycastr`, and I launched it. It doesnt show up in the list o apps under Security & Privacy -> Accessibility so i cant enable it. I am running Mac OSX 10.13.1.

I also did brew cask reinstall keycastr but same issue. No error is show on install.

Any ideas? Thanks.

**Comment by Brendan-Schuetze** [comment]
I'm also having this problem. Also on 10.13.1

**Comment by akitchen** [comment]
You will need to unlock the settings pane and drag KeyCastr into the list yourself. `brew cask install` is not able to do this for you, per the message it emits:

```
→ brew cask install keycastr
==> Creating Caskroom at /usr/local/Caskroom
==> We'll set permissions properly so we won't need sudo in the future
Password:
==> Satisfying dependencies
==> Downloading https://github.com/keycastr/keycastr/releases/download/v0.9.5/KeyCastr.app.zip
######################################################################## 100.0%
==> Verifying checksum for Cask keycastr
==> Installing Cask keycastr
==> Moving App 'KeyCastr.app' to '/Applications/KeyCastr.app'.
==> Enabling accessibility access
Warning: Accessibility access cannot be enabled automatically on this version of macOS.
See System Preferences to enable it manually.
🍺  keycastr was successfully installed!
```
When KeyCastr runs without the permissions it needs it also prompts you to enable it.

**Comment by Brendan-Schuetze** [comment]
It's not clear that you can drag and drop items into the accessibility pane. I would recommend making that message more clear. To me, enabling it manually would mean that it would show up in the accessibility pane, but that i would have to check it off like Dropbox or other apps that use these features. If you don't update these warnings to be a little clearer, people will continue to ask you about this problem.

**Comment by akitchen** [comment]
I agree, the message could be more specific. FYI, you can either drag & drop or click the + button and browse to the location of the app & select it. 

Dropbox is using a hack which subverts Apple’s security model, and they have a lot of resources to play the game of cat & mouse with Apple from release to release of macOS as things change. Google ‘Dropbox Mac security hack’ for more info.

If you know of any open source apps which automatically add themselves to that menu we’d love to hear about them and look into how they’re doing it, but it’s not something we’re inclined to spend a lot of time exploring at the moment, vs. working on other features and fixes.

**Comment by Brendan-Schuetze** [comment]
Thanks for the clarification Andrew. Makes a lot of sense. I know these
kinds of things can be difficult to implement.

Brendan Schuetze '18
717.413.9163
